packages/shared/src/index.ts

```diff
@@ -80,7 +80,7 @@ export const toTypeString = (value: unknown): string =>
   objectToString.call(value)
 
 export const toRawType = (value: unknown): string => {
-  return toTypeString(value).slice(8, -1)
+  return toTypeString(value).slice('[object '.length, -1)
```

Comment:
I don't think this change is necessary. You can just put the background information in comments.

Reply:
ok